contracts/bridge/Bridge.sol

```diff
@@ -83,74 +85,169 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
         uint64 handleNonce
     );
 
-    // start allows the value transfer request.
-    function start() external onlyOwner {
-        isRunning = true;
+    modifier onlySigners()
```

Comment:
This can be moved into `BridgeMultiSig` if BridgeMultiSig is inherted `Ownable`?